index.html

```diff
@@ -253,9 +253,22 @@ <h4>Skills</h4>
 
 
         </div>
+           <section id="contact-me">
+            <h3 class="contatct-head">Contact me</h3>
+            <p class="contact-desc">If you have an application you <br> are interested in developing, a <br> feature that you
+                need built or a <br> project that needs coding. <br>
+                I'd love to help with it!</p>
+            <form action="https://formspree.io/f/xlekpvwq" method="post">
+                <input type="text" id="fullname" name="fullname" maxlength="30" placeholder="Lorem Ipsum">
+                <input name="Email" id="email" type="email" placeholder="loremipsum@gmail.com">
+                <textarea name="messege" id="messege" cols="30" rows="10"
+                    maxlength="500">write your message here</textarea>
+                <button type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly re-adjust the position of this button to meet the requirements 👍 
![Capture](https://user-images.githubusercontent.com/80458199/230501869-2e53e58b-4fcd-4b39-b3b6-91ee1ac0aaf4.PNG)